### PR TITLE
[redisdb] "Variable referenced before assignment" error solved

### PR DESCRIPTION
**Pull request by elafarge**
(wrong variable name was used).

### CODE REVIEW

**Comment by LeoCavaille** [comment]
@elafarge thanks for the fix.
Can you add one more test to the CI that exercises this code path, we should have caught it if it was the case! You can another commit on top of this one to improve our test suite (see `tests/test_redis.py`).

**Comment by elafarge** [comment]
Sure, I'll do that.

File: tests/test_redis.py [hunk 1]
```diff
@@ -235,6 +235,32 @@ def test_slowlog(self):
...
+            "redis_host:localhost", "redis_port:{0}".format(port)])
+
+
+    def test_custom_slowlog(self):
+        port = NOAUTH_PORT
+        test_key = "testkey"
+        instance = {
+            'host': 'localhost',
+            'port': port,
+            'slowlog-max-len': 129
```
Comment (remh) [inline tests/test_redis.py]: You could set it to a low number e.g. 1

then try to generate at least 2 slow queries but make sure that you are actually getting 1 ?

**Comment by remh** [comment]
Nice, could you squash your commits into a single one, then we'll merge

**Pull request merged**